app.rb

```diff
@@ -38,32 +52,13 @@
 end
 
 post '/portal' do
-  domain = params[:email].partition('@').last
-
-  organizations = WorkOS::Portal.list_organizations(
-   domains: [domain],
-  )
-
-  if organizations.data.empty?
-    organization = WorkOS::Portal.create_organization(
-     domains: [domain],
-     name: domain.partition('.').first,
-    )
-
-    portal_link = WorkOS::Portal.generate_link(
-     intent: 'sso',
-     organization: organization.id
-    )
-
-    redirect portal_link
-  else
-    organization = organizations.data.first
+  payload = {
```

Comment:
All of this will be inside of the Ruby SDK for now, but we're holding off on that for now.